templates/base/head.tmpl

```diff
@@ -64,7 +64,7 @@
 	<link rel="mask-icon" href="{{StaticUrlPrefix}}/img/gitea-safari.svg" color="#609926">
 	<link rel="fluid-icon" href="{{StaticUrlPrefix}}/img/gitea-lg.png" title="{{AppName}}">
 {{if .RequireSimpleMDE}}
-	<link rel="stylesheet" href="{{StaticUrlPrefix}}/vendor/plugins/simplemde/simplemde.min.css">
+	<link rel="stylesheet" href="{{StaticUrlPrefix}}/vendor/plugins/easymde/easymde.min.css">
```

Comment:
Create a CSS chunk here instead of vendoring please.

Reply:
Thought, CSS loaded in JS will not be render-blocking, so it's better to have it chunked in the webpack config (probably direct reference to the file in `node_modules` or a wrapper `.less` file.

Reply:
Sorry I wasn't sure how to do it - knew you'd say.

Reply:
I think loading it both in <head> and JS may attempt to load it twice, need to check.

Reply:
Try this instead:

```diff
diff --git a/web_src/js/easymde.js b/web_src/js/easymde.js
index e2e236678..6308c473a 100644
--- a/web_src/js/easymde.js
+++ b/web_src/js/easymde.js
@@ -1,6 +1,5 @@
 import EasyMDE from 'easymde';
-import 'easymde/dist/easymde.min.css';

 import CodeMirror from 'codemirror/lib/codemirror.js';
 import 'codemirror/addon/mode/loadmode.js';
 import 'codemirror/mode/meta.js';
diff --git a/webpack.config.js b/webpack.config.js
index 7c8d55587..921177475 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -58,8 +58,9 @@ module.exports = {
       resolve(__dirname, 'web_src/js/features/eventsource.sharedworker.js'),
     ],
     'easymde': [
       resolve(__dirname, 'web_src/js/easymde.js'),
+      resolve(__dirname, 'node_modules/easymde/dist/easymde.min.css'),
     ],
     ...themes,
   },
   devtool: false,